pimple.js

```diff
@@ -125,7 +129,7 @@
         this.define('pimple',[],function(){return self.Pimple;});
     }
     //CommonJS
-    if(module && module.exports){
+    if(typeof exports !== 'undefined' && module && module.exports){
```

Comment:
Sorry, I don't remember. I have created my own Pimple repository for faster support and I used it. The Pull Request is dated end of december 2014, your question end of march 2016 ... Cancel this line, I don't remember and I don't see now why.

Have a nice day